tcms_tenants/templates/tcms_tenants/new.html

```diff
@@ -68,6 +62,13 @@
             {% block extra_contents %}
                 {{ form.paid_until }}
             {% endblock %}
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_logo">{% trans "Logo" %}</label>
```

Comment:
There is no element with attribute `id_logo`, remove the `for` attribute b/c it doesn't do anything in this case.